diff --git a/configure/CONFIG_SITE b/configure/CONFIG_SITE
index 03ec6d114..a08c2c84e 100644
--- a/configure/CONFIG_SITE
+++ b/configure/CONFIG_SITE
@@ -139,6 +139,11 @@ HOST_WARN=YES
 #  Must be either YES or NO.
 CROSS_WARN=YES
 
+# Always build for the host architecture in addition to cross targets.
+# This will be forced on for EPICS base to ensure that host tools (i.e. msi) are available.
+#  Must be either YES or NO.
+HOST_BUILD=YES
+
 # Installation directory, if you want Base to be installed into a
 #  different location then uncomment and set this.
 #INSTALL_LOCATION=<fullpathname>
diff --git a/configure/RULES_ARCHS b/configure/RULES_ARCHS
index 4aaa75870..a3e87090a 100644
--- a/configure/RULES_ARCHS
+++ b/configure/RULES_ARCHS
@@ -16,9 +16,22 @@ ACTIONS += install
 ACTIONS += buildInstall
 ACTIONS += runtests tapfiles clean-tests test-results junitfiles
 
+# Force host build for EPICS base and associated modules
+ifeq ($(BASE_TOP),YES)
+HOST_BUILD=YES
+endif
+ifeq ($(PARENT_MODULE),EPICS_BASE)
+HOST_BUILD=YES
+endif
+
+ifeq ($(HOST_BUILD),NO)
+BUILD_ARCHS := $(filter-out $(EPICS_HOST_ARCH),$(BUILD_ARCHS))
+endif
+
 actionArchTargets = $(foreach action, $(ACTIONS), \
     $(addprefix $(action)$(DIVIDER), $(BUILD_ARCHS)))
 
+
 cleanArchTargets = $(addprefix clean$(DIVIDER), $(BUILD_ARCHS))
 
 buildDirs = $(addprefix O., $(BUILD_ARCHS))
@@ -38,10 +51,18 @@ endif
 
 CROSS_ARCHS += $(CROSS1) $(CROSS2)
 
+ifeq ($(HOST_BUILD),NO)
+define DEP_template
+  $(2) :
+  $(1)$(DIVIDER)$(2) : O.$(2)
+endef
+else
 define DEP_template
   $(2) : $(EPICS_HOST_ARCH)
   $(1)$(DIVIDER)$(2) : $(1)$(DIVIDER)$(EPICS_HOST_ARCH) O.$(2)
 endef
+endif
+
 $(foreach action, $(ACTIONS), \
     $(foreach arch, $(CROSS_ARCHS), \
         $(eval $(call DEP_template,$(action),$(arch)))))
